src/topology/algebra/infinite_sum.lean

```diff
@@ -22,9 +22,10 @@ import logic.function algebra.big_operators data.set.lattice data.finset
 
 noncomputable theory
 open lattice finset filter function classical
-open_locale topological_space
-local attribute [instance] classical.prop_decidable -- TODO: use "open_locale classical"
+open_locale topological_space classical
 
+/-- given an element of `a : option α`, a default element `b : β` and a function `α → β`, apply this
+function to `a` if it comes from `α`, and return `b` otherwise. -/
 def option.cases_on' {α β} : option α → β → (α → β) → β
```

Comment:
I think this should be moved to `data/option`.

Reply:
Done